roles/kubernetes/master/tasks/kubeadm-upgrade.yml

```diff
@@ -14,7 +14,12 @@
   # Retry is because upload config sometimes fails
   retries: 3
   when: inventory_hostname == groups['kube-master']|first
-  failed_when: kubeadm_upgrade.rc != 0 and "field is immutable" not in kubeadm_upgrade.stderr
+  failed_when: >-
+    kubeadm_upgrade.rc != 0 and
+    "field is immutable" not in kubeadm_upgrade.stderr and
+    "WARNING: failed to determine to backup kube-apiserver cert and key:
```

Comment:
this looks bad

Reply:
@woopstar any further suggestions to handle it?